### ISSUE TITLE
Fader not representing values liner

### ISSUE THREAD

**Issue by torment78** [nhielost/obs-midi-mg#146]
Hello 
Having a issue with faders conected to the OBS 
volume control
Im using bome network form one pc to a nother 
And on the receiving pc i have a instance of bome midi translator one of the channels i use it just to monitor the incoming midi and reading the signal it is sending 0 to 127 linear 
But for some reason in obs the volume is reacting like if i set up a velocity curve 
If i set ur plug in to midi and % then it moves slower the closer  to 0 and when going down it start moving faster and around -35db it jumps to -inf
And if i set it to db it does not jump but it accelerate from -inf up to 0 in a such masive manner all my faders ar almost at max while the volume lvl is set to -20

**Comment by phantom-GH** [comment]
Having the same problem. Were you able to figure it out?

**Comment by torment78** [comment]
Sorry not yet .also the older version dident use to be like this

**Comment by phantom-GH** [comment]
Bummer to hear. Well, fingers crossed it gets fixed at some point lol.

**Comment by phantom-GH** [comment]
Also, would you happen to have an alternative? Wonder about using another program to do it and feed that into OBS?

**Comment by torment78** [comment]
> This may be an OBS issue. Ideally the Midi value should map to a visual position percentage of the level control. Since the OBS level controls (rightly) have a fader law curve to them, 50% travel is -20dB, 25% is around -50dB.
> 
> What's happening is that even if percentage adjustment control is selected, this is applied to the linear gain value, such that 50% is -6dB, 25% is -12dB. This bunches all the control up in the top of the travel.
> 
> If you select dB control, the value is mapped to a dB value which bunches the control up in the bottom half of the fader - worse in terms of perceived audio level.

Yeah sounds like it .but if i dont remember wrong obs dident use to do it .im almost serten i use to have them 1 to 1
